.github/ISSUE_TEMPLATE/new_component_proposal_template.md

```diff
@@ -1,16 +1,18 @@
 ## Description
```

Comment:
```suggestion
---
name: Component Proposal
about: Suggest a component addition or revision
title: "\U0001F195"
labels: 'new component'
assignees: ''

---

## Description
```

Reply:
* `"\U0001F195"` is https://emojipedia.org/emoji/🆕/
* I’m not sure whether `labels: 'new component'` will match [the existing `new component` label](https://github.com/primer/components/issues?q=is%3Aopen+is%3Aissue+label%3A%22new+component%22), or if space is a delimiter (in which case `labels: '"new component"'` should work)

Reply:
Oh whoops never mind, I added this before I saw https://github.com/primer/components/commit/ef17a86bf4de399462b373dac7aafc2af2ffe183